MobileWallet/Screens/Amount/AmountViewController.swift

```diff
@@ -0,0 +1,456 @@
+//  AmountViewController.swift
+
+/*
+	Package MobileWallet
+	Created by Semih Cihan on 12.02.2020
+	Using Swift 5.0
+	Running on macOS 10.15
+
+	Copyright 2019 The Tari Project
+
+	Redistribution and use in source and binary forms, with or
+	without modification, are permitted provided that the
+	following conditions are met:
+
+	1. Redistributions of source code must retain the above copyright notice,
+	this list of conditions and the following disclaimer.
+
+	2. Redistributions in binary form must reproduce the above
+	copyright notice, this list of conditions and the following disclaimer in the
+	documentation and/or other materials provided with the distribution.
+
+	3. Neither the name of the copyright holder nor the names of
+	its contributors may be used to endorse or promote products
+	derived from this software without specific prior written permission.
+
+	THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND
+	CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+	OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+	DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+	CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+	SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+	NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+	LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+	HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+	OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+	SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+import UIKit
+
+class AmountViewController: UIViewController {
+
+    private var buttons = [UIButton]()
+    private let continueButton = SendButton(frame: .zero)
+    private let amountLabel = AnimatedBalanceLabel()
+    private let keypadContainerStackView = UIStackView()
+    private let warningView = UIView()
+    private let warningLabel = UILabel()
+    private let warningBalanceLabel = UILabel()
+    private let transactionViewContainer = UIView()
+    private let animationDuration = 0.3
+    private var balanceCheckTimer: Timer?
+    private let transactionFeeLabel = UILabel()
+    private let gemImageString: NSAttributedString = {
+        let gemAttachment = NSTextAttachment()
+        gemAttachment.image = Theme.shared.images.currencySymbol?.colorized(color: UIColor.black)
```

Comment:
No need for this extension function, you can use the tint like in `TransactionViewControllerExtension`
Also don't use a hardcoded color, go with the same color used in the number label as those will always match.

https://github.com/tari-project/wallet-ios/blob/f7171b0757152fa961dfdc10676a9ba0124be49c/MobileWallet/Screens/Home/Transaction/TransactionViewControllerExtension.swift#L113